3.10/release-notes-new-features310.md

```diff
@@ -427,6 +427,53 @@ arangoexport \
   ...
 ```
 
+
+Query changes for decreasing memory usage
+-----------------------------------------
+
+With the parameters mentioned below, queries can execute with storing input
+and intermediate results temporarily on disk for decreasing of memory usage
+when it reaches a certain threshold.
+Note: this feature is experimental and off by default. The query results will
```

Comment:
Remarks should only be in the feature documentation, not the release notes. The option descriptions should probably be removed (but new start options should still need to be mentioned).